obal/__init__.py

```diff
@@ -214,41 +222,41 @@ def ansible_config_path(cls):
         return os.environ.get('OBAL_ANSIBLE_CFG', os.path.join(cls.data_path(), 'ansible.cfg'))
 
 
-def find_playbooks(playbooks_path):
+def find_playbooks(application_config):
```

Comment:
Should this become a method on the `ApplicationConfig`?

Reply:
like `ac.find_playbooks`? maybe.

Reply:
Yes. Previously it was a very clean standalone test but now it's coupled to app config anyway. I also think that finding playbooks is also part of the application config.